### PR TITLE
Readds holoparasites

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Holoparasites are back in the uplink at a price of 16 telecrystals.
experiment: The Chaos holoparasite now causes hallucinations on bumping and ignites on attack, where it previously ignited on bumping and caused hallucinations on attack.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
so either an emag or a stand? sounds good to me

**Comment by Cheridan** [comment]
hello joan was disappointed in the amount of drama generated in this pr so i'd like to say that this is the worst change ever and you should be ashamed
because of this i'm going to sneak into your house rearrange everything in your room so that you can't find things easily for a while :rage4:

**Comment by Iamgoofball** [comment]
>doomguy emotes

how did I not know these were a thing

**Comment by KorPhaeron** [comment]
let me think about it

**Comment by calzilla1** [comment]
Make it 20 tc, an arm and a leg

**Comment by Bawhoppen** [comment]
nah

**Comment by Bawhoppen** [comment]
that nah was regarding calzilla's post

**Comment by optimumtact** [comment]
Disappointed in the lack of drama? well that's easily fixable

Denied, don't bother remaking this pr either.

**Comment by GeneralGoose** [comment]
i'd go for it if it was 21 TC

**Comment by jughu** [comment]
Make it a cargo exclusive and 40 TC 
Gotta be worth if if you SPEND all THOOOSE POINTS ON NULL CRATES

**Comment by tedward1337** [comment]
@optimumtact you made the right decision closing this
then the wrong decision opening it again

**Comment by Davdichan** [comment]
"A random assistant can disarm a guy with a double e-sword and kill him with it"

If thats our justification for  why borgs need turbo stuns, then why the hell are we allowing Holos back in the game when they can't removed from a traitor or looted.

**Comment by HankMoodyExperience** [comment]
you all fail to mention the fact that you are reliant upon another player to bail your ass out of any situation, and its a 100% gamble between week old accounts and oldman robustins(with preference to garbage players), all for 80% of your telecrystals.

>tl;dr this is super duper op and should not be in the game because somebody killed me one time and i couldnt disarm them and kill them, fucking syndicate firing pins

**Comment by Bawhoppen** [comment]
you can reset your stand once

**Comment by KorPhaeron** [comment]
also what do we do about protectors tanking bosses forever

**Comment by ChangelingRain** [comment]
kor #19590 will help with that(also you could totally shitcode and make bosses use ex_act(1) on protectors, which will do tons of damage even in shield mode)

**Comment by ChangelingRain** [comment]
alternatively you could make boss damage scale up if it keeps attacking the same target, which would also prevent that

**Comment by Kromgar** [comment]
You would wnat to have that on an interval incase lag causes a dude to get bit and then bites get progressively worse would be terrible

**Comment by WJohn** [comment]
These are simply too good. They're bought just as often as emags, I don't know what the exact stats are but I'd go with probably 50% of traitors buy this thing over all other options.

**Comment by ChangelingRain** [comment]
The buy stats are currently 0% because they can't buy it

**Comment by optimumtact** [comment]
@KorPhaeron @Cheridan needs final decision

**Comment by KorPhaeron** [comment]
I'm not ready to go through this balance argument again and the sprites still irritate me
